infra/main.bicep

```diff
@@ -258,7 +256,7 @@ module searchService 'core/search/search-services.bicep' = {
     name: !empty(searchServiceName) ? searchServiceName : 'gptkb-${resourceToken}'
     location: !empty(searchServiceLocation) ? searchServiceLocation : location
     tags: tags
-    publicNetworkAccess: publicNetworkAccess
+    publicNetworkAccess: publicNetworkAccess == 'Enabled' ? 'enabled' : 'disabled'
```

Comment:
I would prefer to have a compare that isn't case sensitive.  lower(publicNetworkAccess) before the compare?